src/builder/copy.ts

```diff
@@ -0,0 +1,54 @@
+import { promises as fsp } from "node:fs";
+import { relative, resolve } from "pathe";
+import { globby } from "globby";
+import { symlink, rmdir, warn } from "../utils";
+import type { CopyBuildEntry, BuildContext } from "../types";
+
+const copy = fsp.cp || fsp.copyFile;
+
+export async function copyBuild(ctx: BuildContext) {
+  const entries = ctx.options.entries.filter(
+    (e) => e.builder === "copy",
+  ) as CopyBuildEntry[];
+  await ctx.hooks.callHook("copy:entries", ctx, entries);
+  for (const entry of entries) {
+    const distDir = entry.outDir!;
+    if (ctx.options.stub) {
+      await rmdir(distDir);
```

Comment:
If one doesn't specify a subdir for `outDir`, then this will delete the whole `dist` dir, right? In my use case, I just want to copy a single file to the root of the `dist` folder.

Reply:
Thats true. We should not allow this or at least use a prompt to confirm action